

#Coordinates of 4 corners of the detected ROI
#float32 x
#float32 y
#float32 h
#float32 w

float32[] bbox
mask_point[] mask

float64 confidence # score of the detection

string object_class #detected class

